web_src/js/features/common-issue.js

```diff
@@ -58,16 +57,9 @@ export function initCommonIssue() {
       // checkboxes stay checked after reload
       if (action === 'close' || action === 'open') {
         // uncheck all checkboxes
-        $('.issue-checkbox input[type="checkbox"]').each((_, e) => { e.checked = false });
+        $('.issue-checkbox').each((_, e) => { e.checked = false });
       }
       window.location.reload();
     });
   });
-
-  // NOTICE: This event trigger targets Firefox caching behaviour, as the checkboxes stay
-  // checked after reload trigger checked event, if checkboxes are checked on load
-  $('.issue-checkbox input[type="checkbox"]:checked').first().each((_, e) => {
-    e.checked = false;
-    $(e).trigger('click');
-  });
```

Comment:
This seems no longer neccesary, and in fact it caused an issue where all checkboxes were being checked after page reload.